### ISSUE TITLE
[Feature request]: Framerate adjustment options

### ISSUE THREAD

**Issue by marek-lach** [ShadowBlip/OpenGamepadUI#414]
Just like with SteamOS, or [this project](https://private-user-images.githubusercontent.com/14330834/246607517-ec35272e-4c9e-4386-9b0f-3b4e3aa0cf6d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzExMDI0MTgsIm5iZiI6MTczMTEwMjExOCwicGF0aCI6Ii8xNDMzMDgzNC8yNDY2MDc1MTctZWMzNTI3MmUtNGM5ZS00Mzg2LTliMGYtM2I0ZTNhYTBjZjZkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA4VDIxNDE1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk3ZTQ4NDU2Y2U1OTMyZDgxMjE4NjExZTZjMzcwYzlhYzFmNjQ2NTgyNWVkZTUxNmFjMTVmYWJjZmNlODY3YmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.l6dPX5lSdSDukyrMTqNiC-xHOyyH9jAdt-ZYJvNUcps), OpenGamepadUI could have options to set game framerate limit at e.g. 20, 30, or 60 FPS.

Thank you,

**Comment by ShadowApex** [comment]
Yes, that PR would include the option to do framerate limiting, but it needs testing and other considerations before it is merged.
